Sources/Swift/Integrations/UserFeedback/SentryUserFeedbackForm.swift

```diff
@@ -334,13 +334,15 @@ class SentryUserFeedbackForm: UIViewController {
         button.backgroundColor = config.theme.submitBackground
         button.setTitleColor(config.theme.submitForeground, for: .normal)
         button.addTarget(self, action: #selector(submitFeedbackButtonTapped), for: .touchUpInside)
+        button.accessibilityIdentifier = "io.sentry.feedback.form.submit"
         return button
     }()
     
     lazy var cancelButton = {
         let button = UIButton(frame: .zero)
         button.setTitle(config.formConfig.cancelButtonLabel, for: .normal)
         button.accessibilityLabel = config.formConfig.cancelButtonAccessibilityLabel
+        button.accessibilityIdentifier = "io.sentry.feedback.form.cancel"
```

Comment:
Thanks for adding the accessibilityIdentifiers 👍

Reply:
Yep it was a good call, esp for the buttons since those labels can be changed in the configs 👍🏻